src/util/existing-workspace-folder-pick.ts

```diff
@@ -0,0 +1,19 @@
+/*-----------------------------------------------------------------------------------------------
+ *  Copyright (c) Red Hat, Inc. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE file in the project root for license information.
+ *-----------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+
+export class ExistingWorkspaceFolderPick {
```

Comment:
As a Util class this really should have TS/JS docs since it is used as an API for the rest of the code.